examples/platform/silabs/efr32/rs911x/hal/efx32_ncp_host.c

```diff
@@ -113,15 +115,15 @@ static void efx32_spi_init(void)
     // Enable USART interface pins
     GPIO->USARTROUTE[SPI_USART_ROUTE_INDEX].ROUTEEN = GPIO_USART_ROUTEEN_RXPEN | // MISO
         GPIO_USART_ROUTEEN_TXPEN |                                               // MOSI
-        GPIO_USART_ROUTEEN_CLKPEN | GPIO_USART_ROUTEEN_CSPEN;
+        GPIO_USART_ROUTEEN_CLKPEN;                                               //| GPIO_USART_ROUTEEN_CSPEN;

```

Comment:
`GPIO_USART_ROUTEEN_CSPEN` if not required do not add as comment

Reply:
done